src/framework/midi/midievent.h

```diff
@@ -767,11 +767,13 @@ struct Event {
                 break;
             //D3.3
             case Opcode::ControlChange: {
-                std::set<uint8_t> skip = { 6, 38, 98, 99, 100, 101 };
-                if (skip.find(index()) == skip.end()) {
```

Comment:
If current controller number (`index()`) is not in in the _skip_ set then break the _Opcode::ControlChange_ case immediately, resulting in keeping _0_ from the `Event` initialization as the controller number. Therefore removed, because other controller numbers _are_ handled by the `default` case.
A new MIDI event always goes through `toMidi20` after reception and being created as a Midi 1.0 event in the first place.